src/components/sds/Textarea/index.tsx

```diff
@@ -81,6 +81,7 @@ const StyledTextInput = styled.TextInput<Pick<StyledProps, "$fieldSize">>`
     px(TEXTAREA_SIZES[$fieldSize].paddingHorizontal)};
   padding-vertical: ${({ $fieldSize }: Pick<StyledProps, "$fieldSize">) =>
     px(TEXTAREA_SIZES[$fieldSize].paddingVertical)};
+  font-family: "Inter-Regular";
```

Comment:
Nice catch!

But I believe we'll need to do something like the below here so we mimic what's being done in Typography for consistency:
```
  font-family: ${Platform.select({
    ios: "Inter-Variable",
    android: "Inter-Regular",
  })};
  font-weight: ${Platform.select({
    ios: "400",
    android: "normal",
  })};
```

The reason being `iOS` is using the `Inter-VariableFont_opsz,wght.ttf` file and `Android` is using the other font files.

I believe we'll actually need to apply this same fix for the `Input` component, would you mind double checking it? Thanks!